playbooks/roles/mattermost/templates/config.json.j2

```diff
@@ -164,38 +186,37 @@
         "EnableSignInWithEmail": true,
         "EnableSignInWithUsername": true,
         "SendEmailNotifications": true,
-        "UseChannelInEmailNotifications": false,
```

Comment:
This line seems to have been removed?

Reply:
Thank you. I missed it. I've added it now.